### DISCUSSION TITLE
Can gumtree diffs be displayed in a terminal?

### DISCUSSION THREAD

**Discussion by ideasman42** [GumTreeDiff/gumtree#288] [Q&A] [answered]
Since "Getting Started" only mentions webdiff, I didn't see any information about using gumtree to display diffs on the command line.

----

For some context, I was curious to compare gumtree to delta (for e.g.) to see if this could be added as a preset to this package: https://codeberg.org/ideasman42/emacs-diff-ansi

**Comment by jrfaller** [comment, marked as answer]
Hi!

There is the `textdiff` command for serialized diffs, but it's not designed for end-users to read.

It would be great to have a console diff, but there are some caveats. Representing update and move in an inline diff is an open problem! Therefore I use a side to side diff to visualize the diff in GumTree. However in delta there are side-to-side diffs, so that could work. Another problem with move enabled diffs is that I am not sure it is compatible with synchronized scrolling. In GumTree I solved this by allowing to scroll independently the two files and allowing user to click on actions to scroll automatically the other files. This one is tough for a term based output.

I hope this help!

Cheers.
